geomesa-nifi-processors/geomesa-nifi-processors-accumulo/src/test/resources/application.conf

```diff
@@ -0,0 +1,32 @@
+geomesa = {
```

Comment:
Hmm....  I have a suggestion that I both like and hate.

How about a common GeoMesa test jar?

On the plus side, we wouldn't have to copy this config file a zillion times across sub-modules and projects.

On the downside, we'd have one place where we could break this build without knowing it...

Thoughts?  (Since we are talking about the GeoMesa-HBase-DataStore test jar, this seems apropos.

Reply:
that seems related to our other idea of building up a suite of data-store tests in index-api that could be run in each back-end. seems like it could be a good idea, and if we expose the underlying bits well enough we could plug it into nifi here too. if the tests and the data are aligned, then we wouldn't have to worry about the data changing out from under us.